client/jetpack-connect/controller.js

```diff
@@ -123,7 +114,8 @@ export function maybeOnboard( { query, store }, next ) {
 
 export function newSite( context, next ) {
 	analytics.pageView.record( '/jetpack/new', 'Add a new site (Jetpack)' );
-	jetpackNewSiteSelector( context );
+	removeSidebar( context );
+	context.primary = <JetpackNewSite path={ context.path } locale={ context.params.locale } />;
```

Comment:
I'm wondering if I'm in a nitpicky enough mood to ask you to 🔤 these when they're on the same line 🤔 :trollface:

Reply:
💥 98c3900715967788592eea3b0f998f4d6347d294